cmd/telemetrygen/internal/traces/worker.go

```diff
@@ -53,8 +59,11 @@ func (w worker) simulateTraces() {
 		ctx, sp := tracer.Start(context.Background(), "lets-go", trace.WithAttributes(
 			attribute.String("span.kind", "client"), // is there a semantic convention for this?
 			semconv.NetPeerIPKey.String(fakeIP),
-			semconv.PeerServiceKey.String("telemetrygen-server"),
+			semconv.PeerServiceKey.String(w.serviceName),
 		))
+		for j := 0; j < w.loadSize; j++ {
+			sp.SetAttributes(attribute.String(fmt.Sprintf("load-%v", j), string(make([]byte, charactersPerMB))))
```

Comment:
This attribute is so that we can test traces with large payloads, which is something we needed in testing the max receive size of the OTLP receiver gRPC endpoint. For example, we can set loadSize to 3 and it will generate a trace of size 3 MB which will be accepted by the receiver. Or we can set loadSize to 5 and it will generate a trace of size 5 MB which will not be accepted by the receiver with the default max size.

Reply:
Since the user can see this attribute, I would add this to the description of the `size` option.